inifile.cc

```diff
@@ -109,7 +105,7 @@ inifile_load_file(gbfile* fin, inifile_t* inifile, const char* myname)
     char* cin = lrtrim(buf);
```

Comment:
If you just make cin a QString(buf).trimmed(), I think we save some xstrdup chatter below.